src/block/countdown/edit.js

```diff
@@ -0,0 +1,511 @@
+
+	const labelTextClasses = getTypographyClasses( attributes, 'label%s' )
+
+	const messageTextClasses = getTypographyClasses( attributes, 'message%s' )
+
+	const blockClassNames = classnames( [
+		className,
+		'stk-block-countdown',
+	] )
+
+	const contentClassNames = classnames( [
+		'stk-block-countdown__content-container',
+	] )
+
+	const dayDigitClassNames = classnames( [
+		'stk-block-countdown__digit',
+		'stk-block-countdown__digit_day',
+		digitTextClasses,
+	] )
+
+	const hourDigitClassNames = classnames( [
+		'stk-block-countdown__digit',
+		'stk-block-countdown__digit_hour',
+		digitTextClasses,
+	] )
+
+	const minuteDigitClassNames = classnames( [
+		'stk-block-countdown__digit',
+		'stk-block-countdown__digit_minute',
+		digitTextClasses,
+	] )
+
+	const secondDigitClassNames = classnames( [
+		'stk-block-countdown__digit',
+		'stk-block-countdown__digit_second',
+		digitTextClasses,
+	] )
+
+	const dayLabelClassNames = classnames( [
+		'stk-block-countdown__label_day',
+		'stk-block-countdown__label',
+		labelTextClasses,
+	] )
+
+	const hourLabelClassNames = classnames( [
+		'stk-block-countdown__label_hour',
+		'stk-block-countdown__label',
+		labelTextClasses,
+	] )
+
+	const minuteLabelClassNames = classnames( [
+		'stk-block-countdown__label_minute',
+		'stk-block-countdown__label',
+		labelTextClasses,
+	] )
+
+	const secondLabelClassNames = classnames( [
+		'stk-block-countdown__label_second',
+		'stk-block-countdown__label',
+		labelTextClasses,
+	] )
+
+	const messageClassNames = classnames( [
+		'stk-block-countdown__message',
+		messageTextClasses,
+	] )
+
+	useEffect( () => {
+		if ( attributes.countdownType === 'recurring' ) {
+			setAttributes( { actionOnExpiration: '' } )
+		}
+	}, [ attributes.countdownType ] )
+
+	return (
+		<>
+			{ isSelected && (
+				<>
+					<InspectorTabs />
+					<InspectorStyleControls>
+						<PanelAdvancedSettings
+							title={ __( 'General', i18n ) }
+							id="countdown"
+							initialOpen={ true }
+						>
+							<AdvancedToolbarControl
+								controls={ COUNTDOWN_TYPE_OPTIONS }
+								attribute="countdownType"
+								value={ attributes.countdownType }
+								fullwidth={ true }
+								default={ 'dueDate' }
+								isSmall={ false }
+							/>
+							<span>
+								{ attributes.countdownType === 'dueDate' ? 'End Date' : 'Start Date' }
+							</span>
+							<DateTimePicker
+								currentDate={ attributes.date }
+								is12Hour={ true }
+								onChange={ currentDate => {
+									// Do not include seconds
+									setAttributes( { date: currentDate.slice( 0, currentDate.length - 3 ) } )
+								} }
+								__nextRemoveResetButton={ true }
+							/>
+							{	attributes.countdownType === 'dueDate' && (
+								<>
+									<AdvancedSelectControl
+										label={ __( 'Timezone', i18n ) }
+										value={ attributes.timezone }
+										options={ TIMEZONE_OPTIONS }
+										defaultValue={ '' }
```

Comment:
Please remove the curly brackets for plain string values, please apply this to the others too.
```suggestion
										defaultValue=""
```